.gitignore

```
@@ -2,7 +2,7 @@
 node_modules/**/*
 .serverless_nextjs
 .env
-*.DS_Store
+**/.DS_Store
```

This also works, which should include any file:
```suggestion
.DS_Store
```